tests/Integration/Behaviour/Features/Scenario/Order/order_odd_tax.feature

```diff
@@ -11,6 +12,12 @@ Feature: Order from Back Office (BO)
     Given email sending is disabled
     And the current currency is "USD"
     And country "FR" is enabled
+    And the module "dummy_payment" is installed
```

Comment:
Why did you move this? Careful I don't remember exactly why but I believe the order of these creations was very specific and all was placed in a specific order for some good reason 😅